### PR TITLE
Use DALI_extra in loader test

### PR DESCRIPTION
**Pull request by klecki**
Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

#### Why we need this PR?
- Refactoring test to use only DALI_extra

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: dali/pipeline/operators/reader/loader/loader_test.cc [hunk 1]
```diff
@@ -34,18 +34,16 @@ class DataLoadStoreTest : public DALITest {
 };
 
 typedef ::testing::Types<CPUBackend> TestTypes;
-string loader_test_image_folder = "/data/dali/benchmark";  // NOLINT
+string loader_test_image_folder = testing::dali_extra_path() + "/db/single/jpeg";  // NOLINT
 
 TYPED_TEST_SUITE(DataLoadStoreTest, TestTypes);
 
-const char* path = std::getenv("DALI_TEST_CAFFE_LMDB_PATH");
```
Comment (JanuszL) [inline dali/pipeline/operators/reader/loader/loader_test.cc]: Remove this from `https://github.com/NVIDIA/DALI/blob/1f63dfa105112b5ab34d8ccb8be88d90020da7dc/qa/L0_self-test/test.sh#L23` as well.

Comment (JanuszL) [reply dali/pipeline/operators/reader/loader/loader_test.cc]: And update https://github.com/NVIDIA/DALI/blob/master/dali/benchmark/caffe_alexnet_bench.cc#L43 as well.

Comment (klecki) [reply dali/pipeline/operators/reader/loader/loader_test.cc]: I don't won't to change the benchmark data, those are not exactly tests, but are meant to measure perf on the original dataset.

Comment (klecki) [reply dali/pipeline/operators/reader/loader/loader_test.cc]: Updated the L0.

**Comment by klecki** [comment]
!build

**Pull request merged**